com.unity.render-pipelines.universal/Shaders/Lit.shader

```diff
@@ -129,6 +130,7 @@ Shader "Universal Render Pipeline/Lit"
             //--------------------------------------
             // GPU Instancing
             #pragma multi_compile_instancing
+            #pragma instancing_options renderinglayer
```

Comment:
What's the impact of this if light layers are turned off?

Reply:
This is enabled with instancing. 
renderinglayer will be sent with or without lightlayer enabled. So extra CPU cost when using instancing. Either the feature is used or not. (Note: In HDRP renderinglayer  is used also for decal layers, and in the future it can be used for other purpose).

Reply:
I wasn't sure myself what it did but HDRP was enabling this so I assumed it was also needed by URP rendering layers. There is no option to disable this if the project does not use light layers, is there?
Furthermore, I don't think this is documented feature, as it is not documented in [2021.1 doc](https://docs.unity3d.com/2021.1/Documentation/Manual/GPUInstancing.html) in the valid switches for `#pragma instancing options`.